site/e2e/api.ts

```diff
@@ -150,7 +152,8 @@ export async function verifyConfigFlagArray(
 	);
 
 	// Verify array of options with simple dots
-	for (const item of opt.value) {
+	// biome-ignore lint/suspicious/noExplicitAny: opt.value is any
+	for (const item of opt.value as any) {
```

Comment:
Kind of wish we could've at least written the expression as `const item of opt.value as any[]`, to get some degree of LSP feedback. I don't think it's a huge deal, since this is a test file, but yeah, we'll definitely need to clean up our types later on